skimage/color/tests/test_delta_e.py

```diff
@@ -137,6 +138,20 @@ def test_cmc():
 
     assert_allclose(dE2, oracle, rtol=1.e-8)
 
+    # Test that equal colors have a distance of 0:
+    lab1 = lab2
+    assert_equal(deltaE_cmc(lab1, lab2), np.zeros(oracle.shape))
+    # If they're *almost* equal, than that's still fine:
+    lab2[0, 0] = lab2[0, 0] + np.finfo(float).eps
+    assert_almost_equal(deltaE_cmc(lab1, lab2), np.zeros(oracle.shape))
+
+    # Generalize to the single item case:
+    lab1 = lab2 = np.array([0., 1.59607713, 0.87755709])
+    assert_equal(deltaE_cmc(lab1, lab2), 0)
+
+    #lab2 = np.array([0.+ np.finfo(float).eps, 1.59607713, 0.87755709])
```

Comment:
Is it intended that this is commented? If yes, then add a comment why. If it is not needed any more, then remove the two lines. Thanks!